public/js/controllers/home-controller.js

```diff
@@ -120,4 +121,71 @@ function($scope, $http, planService, notificationService) {
             });
         });
     };
+
+    $scope.hasPrereqs = function(course) {
+
+        //This is ugly
```

Comment:
Block comments in general are a pain to work with. In many IDEs, multi line comments done in this fashion are easily toggled via a keybinding such as F7. which will append or remove a "//" to the start of each line. Even in vim, comments done this way can be inserted or deleted using block highlighting (ctrl+v, highlight all the lines and press x to delete, or press (capital)I and then type //, then [esc])

On the other hand, block comments using the /* */ notation require you to locate both the opening /* and the closing */, and delete both of them by hand. Additionally, if you put a /* or */ inside of another (ie trying to comment out a large number of functions in a java file which uses javadoc syntax using this method) the behavior will not be as you desired.

`/**
  * This is my rad function
  */
public static void main() { ... }`
If you want to comment out this function, perhaps you would do this:
`
/*
/**
  * This is my rad function
  */
public static void main() { ... }
*/`
But this does not work, because the first */ terminates both comments.

Thus, my opinion is that the // style of commenting is superior in most situations